### PR TITLE
Fix Foundations overview tiles

### PR DESCRIPTION
**Pull request by amritadesmet**
Overview page tiles need fixing in association with #4392 

- Add new tiles for Shadow, Color, Iconography 
- Create consistent color order of the tiles

### CODE REVIEW

**Comment by amritadesmet** [comment]
Not sure how I committed 5 past commits, only wanted to commit the tiles portion...

**Comment by joshwooding** [comment]
How does this PR related to #4391, does this replace it or should the change not include those ones.

**Comment by joshwooding** [comment]
> Not sure how I committed 5 past commits, only wanted to commit the tiles portion...

This would happen if you branched off that work, you have to branch off main when you want a fresh piece of work.

File: site/docs/foundations/index.mdx [hunk 1]
```diff
@@ -20,30 +30,40 @@ Density relates to the amount of content that can fit on a screen based on the s
   </OverviewCard>
 
 <OverviewCard
-    icon={<Image src="/img/foundations/duration.svg" alt="duration"  />}
+    icon={<Image src="/img/foundations/duration2.svg" alt="duration"  />}
```
Comment (origami-z) [inline site/docs/foundations/index.mdx:33]: do you meant to replace the existing svg? if so, add `*2` will leave unused files behind

**Review by Fercas123 — DISMISSED** [review]

**Review by joshwooding — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**